tagstudio/src/core/utils/missing_files.py

```diff
@@ -19,6 +19,7 @@ class MissingRegistry:
     library: Library
     files_fixed_count: int = 0
     missing_files: list[Entry] = field(default_factory=list)
+    library_file_cache: set[Path] = None
```

Comment:
#608 already added a library file cache [here](https://github.com/TagStudioDev/TagStudio/blob/b72a2f233141db4db6aa6be8796b626ebd3f0756/tagstudio/src/core/library/alchemy/library.py#L140). Could you leverage that one instead of adding a new one?